.github/workflows/cicd-pipeline.yml

```
@@ -0,0 +1,351 @@
+          IFS='.' read -r -a V <<< "$LAST_TAG_NUM"
+          # Incrementa o MINOR e reseta o PATCH
+          V[1]=$((V[1] + 1))
+          V[2]=0
+          NEW_VERSION="v${V[0]}.${V[1]}.${V[2]}"
+          BRANCH_NAME="release/$NEW_VERSION"
+          echo "Creating new branch from develop: $BRANCH_NAME"
+          git checkout -b $BRANCH_NAME
+          git push origin $BRANCH_NAME
+
+      - name: Delete Merged Feature Branch
+        run: |
```

actionlint (1.7.8)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)